test/scripts/start-services.sh

```diff
@@ -35,10 +35,15 @@ start_geth() {
 
     local data_dir="$output_dir/geth"
 
+    # removes TTD bomb for when beacon chain is not used
+    if [ "$start_beacon_sync" == "false" ]; then
+        jq 'del(.config.terminalTotalDifficulty)' config/genesis.json > "$output_dir/genesis.json"
```

Comment:
This will override the difficulty config from lines 27-34. I think you should use:
```
... "$output_dir/genesis.json" | sponge "$output_dir/genesis.json"
```

Reply:
Good point! Addressed in https://github.com/Snowfork/snowbridge/pull/666/commits/fea4e9963bfe684414a2fe6f7b22a08b422decd4.

Although interestingly `$output_dir/genesis.json` wasn't actually used before.

Reply:
Ok tested and it does not hang on contract deploy anymore.